docs/ACCEPTANCE_TESTS.md

```diff
@@ -0,0 +1,97 @@
+# Rule acceptance tests 
+
+## Goal
+Execute the validator against all datasets (latest version hosted in the MobilityDatabase) and quantify the effect of a code change on all of them.  
+The source of truth is defined as the latest stable version of the validator (the on available on [the master branch of this repository](https://github.com/MobilityData/gtfs-validator/tree/master)). 
+
+For the latest version of all GTFS datasets from the MobilityArchives, the validation report from both the snapshot and the source of truth are compared. An acceptance test report is generated: it quantifies for each agency/dataset the number of new errors (as defined [here](https://github.com/MobilityData/gtfs-validator/blob/master/RULES.md#definitions)) in introduced.
+   
+## Process description
+
+![steps](https://user-images.githubusercontent.com/35747326/120213646-6fb7bb80-c201-11eb-8520-39c88e9753d0.png)
```

Comment:
The diagram says "reference v2.0", but I think that's outdated now and you're just using the main branch instead?

Reply:
Indeed, thanks. Here is the updated diagram that to be integrated to `ACCEPTANCE_TESTS.m
![Blank Diagram (14)](https://user-images.githubusercontent.com/35747326/139877746-fd047437-38b3-44fa-aeb8-37d925c289e8.png)
d`